schemas/individual_response/definitions.json

```
@@ -0,0 +1,22 @@
+{
+  "$id": "https://eq.ons.gov.uk/individual_response/definitions.json",
+  "individual_response": {
+    "type": "object",
+    "properties": {
+      "enabled": {
+        "type": "boolean",
+        "description": "Controls whether the individual response definition can be shown"
+      },
+      "for_list": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+        "description": "The list to use when determining whether the individual response definition can be shown"
```

We shouldn't describe runner implementation specifics in schema descriptions, and this is consistent with the description for the section id:
```suggestion
        "description": "The list from which an individual response can be requested"
```